### PR TITLE
Test/cta

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#2078 

### Description

I've increased the test coverage for `CTA` so it follows our 80% coverage target.

### CODE REVIEW

File: packages/react/src/components/CTA/__tests__/CTA.test.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import CardCTA from '../CardCTA';
+import CTALogic from '../CTALogic';
+import FeatureCTA from '../FeatureCTA';
+import { mount } from 'enzyme';
+import React from 'react';
+import TextCTA from '../TextCTA';
+
+describe('CTA', () => {
+  it('Renders TextCTA | type jump', () => {
+    CTALogic.jump = jest.fn();
```
Comment (asudoh) [inline packages/react/src/components/CTA/__tests__/CTA.test.js:18]:
```suggestion
    jest.spyOn(CTALogic, 'jump');
```

File: packages/react/src/components/CTA/__tests__/CTA.test.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    const cta = mount(<TextCTA type="jump" href="#" copy="Lorem Ipsum" />);
+    cta.find('a').simulate('click');
+    expect(CTALogic.jump).toHaveBeenCalled();
+  });
+
+  it('Renders TextCTA | type video', () => {
+    CTALogic.setLightBox = jest.fn();
```
Comment (asudoh) [inline packages/react/src/components/CTA/__tests__/CTA.test.js:25]:
```suggestion
    jest.spyOn(CTALogic, 'setLightBox');
```

File: packages/react/src/components/CTA/__tests__/CTA.test.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    const cta = mount(
+      <TextCTA type="video" href="#" videoTitle={[{ title: 'Lorem Ipsum' }]} />
+    );
+    cta.find('a').simulate('click');
+    expect(CTALogic.setLightBox).toHaveBeenCalled();
+  });
+
+  it('Renders FeatureCTA | type video', () => {
+    CTALogic.setLightBox = jest.fn();
+    CTALogic.launchLightBox = jest.fn();
```
Comment (asudoh) [inline packages/react/src/components/CTA/__tests__/CTA.test.js:35]:
```suggestion
    jest.spyOn(CTALogic, 'setLightBox');
    jest.spyOn(CTALogic, 'launchLightBox');
```

File: packages/react/src/components/CTA/__tests__/CTA.test.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+        videoTitle={[{ title: 'Lorem Ipsum' }]}
+      />
+    );
+    cta.find('a').simulate('click');
+    expect(CTALogic.launchLightBox).toHaveBeenCalled();
+  });
+
+  it('Renders CardCTA | type video', () => {
+    CTALogic.setLightBox = jest.fn();
+    CTALogic.launchLightBox = jest.fn();
```
Comment (asudoh) [inline packages/react/src/components/CTA/__tests__/CTA.test.js:61]:
```suggestion
    jest.spyOn(CTALogic, 'setLightBox');
    jest.spyOn(CTALogic, 'launchLightBox');
```

File: packages/react/src/components/CTA/__tests__/CTA.test.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+    const cta = mount(
+      <CardCTA type="video" videoTitle={[{ title: 'Lorem Ipsum' }]} />
+    );
+    expect(CTALogic.launchLightBox).toHaveBeenCalled();
+    cta.find('a').simulate('click');
+    expect(CTALogic.setLightBox).toHaveBeenCalled();
+  });
+
+  it('Renders CardCTA | type video', () => {
+    CTALogic.jump = jest.fn();
```
Comment (asudoh) [inline packages/react/src/components/CTA/__tests__/CTA.test.js:71]:
```suggestion
    jest.spyOn(CTALogic, 'jump');
```

File: packages/react/src/components/CTA/__tests__/CTA.test.js [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+          href: '#',
+        }}
+      />
+    );
+    cta.find('a').simulate('click');
+    expect(CTALogic.jump).toHaveBeenCalled();
+  });
+
+  it('Renders ButtonCTA | type video', () => {
+    CTALogic.setLightBox = jest.fn();
```
Comment (asudoh) [inline packages/react/src/components/CTA/__tests__/CTA.test.js:85]:
```suggestion
    jest.spyOn(CTALogic, 'setLightBox');
```

File: packages/react/src/components/CTA/__tests__/CTA.test.js [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+        type="video"
+        videoTitle={[{ title: 'Lorem Ipsum' }]}
+      />
+    );
+    cta.find('a').simulate('click');
+    expect(CTALogic.setLightBox).toHaveBeenCalled();
+  });
+
+  it('Renders ButtonCTA | type jump', () => {
+    CTALogic.jump = jest.fn();
```
Comment (asudoh) [inline packages/react/src/components/CTA/__tests__/CTA.test.js:108]:
```suggestion
    jest.spyOn(CTALogic, 'jump');
```

**Comment by jeffchew** [comment]
@RaphaelAmadeu looks like there are a bunch of test failures, can you take a look? One thing at a high level, make sure you're using `root` in your unit tests if you're referencing `window`.

**Pull request merged**